client/run-cypress-tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+export ARLO_AUDITADMIN_AUTH0_BASE_URL=https://votingworks-noauth.herokuapp.com
+export ARLO_JURISDICTIONADMIN_AUTH0_BASE_URL=https://votingworks-noauth.herokuapp.com
+
+trap 'kill 0' SIGINT SIGHUP
+./cypress/seed-test-db.sh
+pushd ..
+FLASK_ENV=test ./run-dev.sh &
```

Comment:
This leaves `run-dev.sh` running after cypress has finished, which makes it fail on the second run since the 3000 port is still occupied by the previous process. I have to run `run-dev.sh` again, which fails but gives me the pid of the blocking process, then kill that process, and then I'm free to run it again. Perhaps add `EXIT` to the trap here? (I tried that and it worked from what I could tell.)

Reply:
Hmm yeah I was struggling with this a lot. I think I remember that I had EXIT in there but then it was giving me problems because it was failing on CI (for some reason trapping EXIT made Circle think the test step failed). I'll mess with it some more

Reply:
Ah yeah, I wondered if it might do that. Perhaps trapping a EXIT on a different script?